clustered/client/src/main/java/org/ehcache/clustered/client/config/builders/ClusteringServiceConfigurationBuilder.java

```diff
@@ -90,18 +98,18 @@ public ServerSideConfigurationBuilder expecting() {
    *
    * @throws NullPointerException if {@code unit} is {@code null}
    * @throws IllegalArgumentException if {@code amount} is negative
+   *
+   * @deprecated
    */
+  @Deprecated
   public ClusteringServiceConfigurationBuilder readOperationTimeout(long duration, TimeUnit unit) {
-    return new ClusteringServiceConfigurationBuilder(this, TimeoutDuration.of(duration, unit));
+    Duration readTimeout = Duration.of(duration, toChronoUnit(unit));
```

Comment:
This can now throw.
And again about API stability, if we decide to drop the methods on the configuration class, this one should go as well. Note that I am _not_ advocating that solution, rather making sure we take a consistent attitude.

Reply:
Yes. I'm ok with coherence of course. We just need to decide

Reply:
If this doesn't go we should also think about defining what mixing calls to this and the new method does.

Reply:
Removed as discussed